src/suspect_place_around_search.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+from geopy.distance import vincenty
+import requests
+import math
+
+
+def search_suspect_place(lat, lng):
+    ''' Return a dictonary containt information of a
+        suspect place arround a given position.
+
+        A suspect place is some company that match a suspect keyword
+        in a Google Places search.
+
+        input:
+            - lat : latitude
+            - lng : longitude
+
+        output: closest_suspect_place:
+
+            closest_suspect_place is a dict with the keys:
+                - name : The name of suspect place
+                - latitude : The latitude of suspect place
+                - longitude : The longitude of suspect place
+                - distance : The distance of suspect place and the given lat, lng
+                - address : The address  of suspect place
+                - phone : The phone of suspect place
+                - id : The Google Place ID of suspect place
+                - keyword : The Keyword that generate the place
+                            in Google Place Search
+
+    '''
+
+    GOOGLE_PLACES_API_KEY = 'SOME_GOOGLE_PLACE_API_KEY'
```

Comment:
Would you mind loading that from `config.ini` as in [`src/geocode_addresses.py`](https://github.com/datasciencebr/serenata-de-amor/blob/master/src/geocode_addresses.py#L18)?